spec/index.html

```diff
@@ -820,20 +820,36 @@ <h3><code>rdf:nil</code></h3>
 
 <section id="ch_ttrei">
   <h2>Triple-term based reification</h2>
-  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract 
-    Syntax specification [[RDF12-CONCEPTS]], referring to an RDF triple that 
-    is used as a value, specifically, in the object position of another triple. 
-    Importantly, a triple term does not imply assertion; that is, the triple it 
-    denotes is not necessarily asserted in the RDF graph. This enables the 
-    representation of metadata or commentary about triples that may or may not 
-    be part of the asserted graph content, including potentially contradictory 
-    relationships. For a triple term to be considered an asserted triple, it 
-    must also appear explicitly in an RDF graph as a top-level triple. Otherwise, 
-    it remains unasserted and serves primarily as a reference or subject of metadata.</p>
+  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract Syntax specification 
+    [[RDF12-CONCEPTS]], referring to an RDF triple that is used as a value, specifically, in the object 
+    position of another triple. This use is a reference to a proposition, and importantly, a triple term 
+    does not by itself imply assertion; that is, the triple it denotes is not necessarily asserted in 
+    the RDF graph. For a triple term to also be considered an asserted triple, it must appear explicitly 
+    in the same RDF graph both as an asserted top-level triple and as an unasserted triple term. 
```

Comment:
This sentence is misleading: a triple is asserted if it is an element ("top-level") in the graph. It **does not need** to be also present as a triple term!

I understand that the nuance here is "for a triple term to be considered asserted", and arguably, the previous wording was already problematic, but with this emphasis on "both", I find it very confusing.

Below is a proposed rephrasing which I consider more acceptable. I'm not pushing hard for this specific wording, but *some* rewording is strongly needed.

```suggestion
    the RDF graph. This is only the case if that triple is also asserted in the RDF graph,
    that is, a direct element of that set of triples.
```